pages/pricing/_standard-pricing.html

```diff
@@ -2,45 +2,104 @@
   <div class="gw-pricing-columns__panel panel-body">
     <div class="plans-heading text-center" data-mh="plans-heading">
       <h2 class="no-anchor margin-top-none">Standard</h2>
-      <p class="description" id="standard-plan">Run production-grade infra on AWS.<br>Ideal for individual teams.</p>
+      <p class="description" id="standard-plan">
+        Run production-grade infra on AWS.<br />Ideal for individual teams.
+      </p>
     </div>
-    <hr>
+    <hr />
```

Comment:
in HTML5 we don't need `/` in `<hr>` or `<br>` please check other examples in project but I think it can be as `<hr>`